tasks/validate.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+# module errors validating
+module Validate
+  # validate for errors
+  class FieldError < StandardError
```

Comment:
errors shouldn't be under `Validate` namespace
move `FieldError` and `WrongNumber` to `Error` module

Reply:
done